sdk-lib/src/main/java/cash/z/ecc/android/sdk/internal/TypesafeDerivationToolImpl.kt

```diff
@@ -21,14 +20,14 @@ internal class TypesafeDerivationToolImpl(private val derivation: Derivation) :
     override suspend fun deriveUnifiedSpendingKey(
         seed: ByteArray,
         network: ZcashNetwork,
-        account: Account
-    ): UnifiedSpendingKey = derivation.deriveUnifiedSpendingKey(seed, network, account)
+        accountIndex: Int
```

Comment:
Again, in Swift this uses the `Zip32AccountIndex` wrapper.

Reply:
Done